File: removeelement.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int retsize = nums.size();
        for (size_t i = 0; i < nums.size(); i++) {
            if (nums[i] == val) {
                nums[i] = 101;
                retsize-=1;
            }
        }
        sort(nums.begin(), nums.end());
        return retsize;
    }
};

File: removeElement.py
def removeEle(nums, val):
    i = 0
    while i < len(nums):
        if nums[i] == val:
            nums.remove(val)
            continue
        i += 1
    return len(nums)
nums = [3,2,2,3]
print(removeEle(nums, 3))
print(nums)
